siriuspy/siriuspy/pwrsupply/bsmp/constants.py

```diff
@@ -12,7 +12,7 @@
 # version of the BSMP implementation of power supplies that is compatible
 # with the current implemenation of this module.
 
-__version__ = 'FRWM - V1.36 V0.41u - 09/04/2020'
+__version__ = 'FRWM - V1.39 V0.44 - 24/06/2022'
```

Comment:
Hi @gabrielbrunheira ! I used this date to conform to the BSMP specification document. I used this date here since this PR is synced with this version (updates only common variables and FBP specific variables). In the second PR (I will make it available soon) the date will be 28/06/2022, according to the latest version of the document. Why 30/06/2022?

Reply:
I thought this date should  match with the PS_Name parameter read from UDC boards. Since it should match the version stated on the BSMP specification, it's ok then to keep 24/06/2022.